src/Standards/Generic/Sniffs/ControlStructures/ControlStructuresBracketsNewLineSniff.php

```diff
@@ -0,0 +1,244 @@
+<?php
+/**
+ * Checks if Control Structures Brackets are on a line by their own
+ *
+ * @author    Greg Sherwood <gsherwood@squiz.net>
+ * @copyright 2006-2018 Squiz Pty Ltd (ABN 77 084 670 600)
+ * @license   https://github.com/squizlabs/PHP_CodeSniffer/blob/master/licence.txt BSD Licence
+ */
+
+namespace PHP_CodeSniffer\Standards\Generic\Sniffs\ControlStructures;
+
+use PHP_CodeSniffer\Sniffs\Sniff;
+use PHP_CodeSniffer\Files\File;
+use PHP_CodeSniffer\Util\Tokens;
+
+class ControlStructuresBracketsNewLineSniff implements Sniff
+{
+
+    /**
+     * The number of spaces code should be indented.
+     *
+     * @var integer
+     */
+    public $indent = 4;
+
+    /**
+     * A list of tokenizers this sniff supports.
+     *
+     * @var array
+     */
+    public $supportedTokenizers = ['PHP'];
```

Comment:
This property isn't needed when you are only checking PHP files. It's the default state, so can be removed.

Reply:
Removed